west.yml

```diff
@@ -104,7 +104,7 @@ manifest:
       revision: 724f7e2a4519d7e1d40ef330042682dea950c991
       path: modules/lib/open-amp
     - name: loramac-node
-      revision: 170a2579dd890f78f5056f0959cdb9c9bea259a1
```

Comment:
This commit isn't needed anymore. We (accidentally) included these changes when the SX126x driver was merged.

Reply:
Ah, yes. Removed.

Reply:
Thanks!